DTML.EduBot/DTML.EduBot.csproj

```diff
@@ -157,6 +157,7 @@
     <Compile Include="LessonPlan\LessonPlanModule.cs" />
     <Compile Include="LessonPlan\Lesson.cs" />
     <Compile Include="LessonPlan\LessonPlan.cs" />
+    <Compile Include="LessonPlan\TestQuestion.cs" />
```

Comment:
Ah, may have been a confusing name but this a test, as in a quiz, for the user to take. Not "test code"